### DISCUSSION TITLE
Alternative Website Support

### DISCUSSION THREAD

**Discussion by Sxri** [3kh0/ext-remover#274] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Hi, I've been pretty active on reading many discussions on this repository, so please, don't take this down yeet or 3kh0.

I'm in search for someone who knows somewhat fluent or better html & css (ATLEAST) and is willing to help me in developing my website "AnchorX". So far we have traffic and people visiting, however, I've been busy lately and need an extra dev to help add games to the website, or modify it in ways they think is good. You'd be listed as an owner, and we'd both work on this project together.

AnchorX official site: https://anchorx.lol

Please, DM lurk#8284 on discord for questions or concerns or obviously if you're interested. Thank you.

**Comment by 3kh0** [comment]
kinda of a L site tbh

**Reply by ghost** [reply]
Bro is mad

**Reply by Sxri** [reply]
yeah no shit. ive been to busy to work on it, literally the point of creating the thread lmao

**Reply by Ironbrodiet** [reply]
still tho the site sucks

**Reply by Sxri** [reply]
> still tho the site sucks

it's not even done yet, its no where near done so ofc it sucks. you sound stupid. i need devs  for it, ive been too busy.

**Reply by Ironbrodiet** [reply]
my bad

**Reply by Gooferball69** [reply]
> 

Holy fudgenuggets he apologize hes automatically better then 90% of people on the internet.

**Comment by ILikeBypassing** [comment]
man just deploy your own holyunblocker

**Reply by Sxri** [reply]
no thats stupid. i'm challenging myself, i like doing work.

**Reply by Ironbrodiet** [reply]
Are any of the games working? I can guarantee putting some of those games into your site will be impossible.

**Reply by Ironbrodiet** [reply]
*Close to impossible

**Comment by Gooferball69** [comment]
jeez why are these guys calling your site bad you are busy with other things

**Reply by Sxri** [reply]
yeah, its okay. I'm just caught up with my outside of my computer life. I had a whole social media account for it that I cant even post on anymore since Ive been so busy, losign followers every minute.

**Comment by Meta0Music** [comment]
I can help you with the site, I know a bit of html/css

**Reply by Sxri** [reply]
Message me on discord please, lurk#8284. 

Thanks
